b2sdk/transfer/emerge/write_intent.py

```diff
@@ -63,6 +63,19 @@ def is_upload(self):
         """
         return self.outbound_source.is_upload()
 
+    def get_large_file_sha1(self):
```

Comment:
3.5 type hint please and generally... maybe we should have a Sha1HexDigest type? It's not just any string that we return here